src/api/useMeasureServiceApi.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import axios from "axios";
+import fail from "jest";
```

Comment:
Might not need this in real code?

Reply:
yeah....I don't even remember putting that there...removed in most recent commit